src/main/java/uk/gov/hmcts/reform/sandl.snlevents/mappers/FactsMapper.java

```diff
@@ -65,6 +72,28 @@ public String mapCreateHearingPartToRuleJsonMessage(CreateHearingPart createHear
     }
 
     public String mapHearingPartToRuleJsonMessage(HearingPart hearingPart) throws JsonProcessingException {
+        return mapDbHearingPartToRuleJsonMessage(hearingPart);
```

Comment:
why 2 methods?

Reply:
it's used in two places for two different reason therefore encapsulated inside of this class.